public/help/comment-preferences.html

```diff
@@ -9,6 +9,6 @@ <h4>Comment Preferences</h4>
   <dd>Enable this option if you prefer not to receive emails for your own comments (for example, when you reply to comments on your own works).</dd>
   <dt>Turn off emails about kudos</dt>
   <dd>Enable this option if you would prefer not to receive email notifications when someone leaves kudos on your work.</dd>
-  <dt>Turn off admin emails</dt>
```

Comment:
Why is this deleted ?

Reply:
The preference was removed 6 years ago in https://otwarchive.atlassian.net/browse/AO3-5043.